interfaces/Portal/.env.example

```diff
@@ -24,10 +24,11 @@ NG_TWILIO_ERROR_CODES_URL=https://www.twilio.com/docs/api/errors
 USE_SSO_AZURE_ENTRA=
 
 # Azure Entra environment-specific IDs.
-# Make sure to set these values the same for the 121-service.
+# Make sure to set these two values the same for the 121-service.
```

Comment:
```suggestion
# Make sure to set this value (AZURE_ENTRA_CLIENT_ID) the same for the 121-service.
```

`AZURE_ENTRA_TENANT_ID` is not referenced or used in the 121-service.

Reply:
I have added AZURE_ENTRA_CLIENT_ID and AZURE_ENTRA_TENANT_ID in 121 service env example file. Now it should be ok because they must be the same.

Reply:
But `AZURE_ENTRA_TENANT_ID` is not used in the 121-service. I would
- remove `AZURE_ENTRA_TENANT_ID` from 121-service
- change this comment to reflect that only this env variable needs to be the same

Otherwise it is confusing that we need to set env variables that are not used.

Reply:
Okay, now I understand and update env files, you can look into it.